designs/administrator/4.7/enable-dynamic-plugin-ui/index.md

```diff
@@ -0,0 +1,58 @@
+
+Admins installing operators which include dynamic plugins will need to opt-in to having their UIs included in the console UI, which can happen as the operator is being configured for install. Trusted operators could perhaps have this option included by default, whereas other not-explicitly-trusted operators would need to have the user explicitly opt in, as there could be a security risk.
+
+
+Once the operator with the dynamic plugin UI is installed, the user can also turn the dynamic plugin UI on or off in the operator details. The operator list view would also convey any operators with disabled dynamic UIs.
+
+## Enabling dynamic plugin UI during operator install
+
+### Trusted operator install
+
+![Operator trusted install](img/1-1-install-rh.png)
+- A new radio group would appear for operators that include dynamic plugin UIs.
+- Trusted operators would have their dynamic UIs enabled by default.
+- A note on “trusted”: Disconnected environments need all operator sources manually added so they all (even Red Hat & certified operators) will appear as “untrusted”.
+
+![Operator install popover](img/1-2-install-rhPop.png)
+- An info popover would further describe implications of enabling an operator’s UI.
+
+### Untrusted operator install
+
+![Operator untrusted install](img/1-3-install-3rd.png)
+- Untrusted operators would have dynamic UIs disabled by default.
+
+![Operator untrusted install alert](img/1-4-install-3rdEnable.png)
+- Enabling the UI for the untrusted operator shows an inline alert conveying implications.
+
+## Enabling/disabling dynamic plugin UIs in operator details
+
+![Operator details setting](img/3-1-details.png)
+- Operators with dynamic plugin UIs would now include a setting in their details that conveys if their dynamic UI is enabled or disabled.
+
+![Operator details modal](img/3-2-details-modal.png)
+- Using the existing editing paradigm, this setting would be launched in a modal.
+- The content of the modal would be the same as the Operator Install page popover and setting.
+
+![Operator details modal enabled](img/3-3-details-modalEnabled.png)
+- The modal would include the same inline alert when enabling UIs for untrusted operators.
+
+![Operator details enabled](img/3-4-detailsEnabled.png)
+- The setting is updated to convey the dynamic plugin UI is enabled.
+
+## Enabling/disabling dynamic plugin UIs in operator details
+
+![Operator list](img/2-1-listView.png)
+- An operator that supports dynamic UIs but does not have them enabled would have an additional status on the Installed Operators page.
+- This status would only appear for users that have ‘read’ access on the operator config and would also be the only users able to update this setting as well.
```

Comment:
Is what you're saying is the user needs `patch` on the `Console` operator in order for the link to appear?  Being able to `get` the `Console` operator would allow us to show any plugins that are available, but not enable/disable them.

Reply:
The link here does not indicate state (Enabled/Disabled).  It seems like it should.  Otherwise the user has to click to find out whether or not the plugin is enabled or disabled.

Reply:
As designed (with the addition of the plugin name when there is more than one)
<img width="1413" alt="Screen Shot 2021-02-09 at 9 31 17 AM" src="https://user-images.githubusercontent.com/895728/107378050-98347200-6ab9-11eb-9cee-7e8d5c030dfe.png">

Perhaps we should follow the pattern from the details page so we can indicate state?
<img width="1060" alt="Screen Shot 2021-02-09 at 9 44 47 AM" src="https://user-images.githubusercontent.com/895728/107379993-789e4900-6abb-11eb-94cf-5bad9ba5d98a.png">